src/app/loadout-builder/generated-sets/GeneratedSets.tsx

```diff
@@ -107,9 +109,7 @@ export default class GeneratedSets extends React.Component<Props, State> {
       <div className={styles.sets}>
         <h2>
           {t('LoadoutBuilder.GeneratedBuilds')}{' '}
-          <span className={styles.numSets}>
-            ({t('LoadoutBuilder.NumStatMixes', { count: sets.length })})
-          </span>
+          <span className={styles.numSets}>({sets.length.toLocaleString()})</span>
```

Comment:
@delphiactual, is this change what you meant?

I did a bit of digging to see if I could use i18n for the number localisation without the need of a key but couldn't figure out a way. The reason I wanted to use i18n is normally I try to keep formatting coming from a single source so that consistency is easier to achieve.